src/LiveComponent/src/Util/DehydratedProps.php

```diff
@@ -113,6 +113,13 @@ public function getNestedPathsForProperty(string $prop): array
         return $nestedPaths;
     }
 
+    public function searchFullPathsForProperty(string $prop): array
```

Comment:
Let's add some tests if possible (with failing / problematic cases, not just happy path)